specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/CreateAVmInWithExtensionsTimeBudget.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
Since you're adding this file, the name CreateAVmInWithExtensionsTimeBudget seems odd. Unless this is a standard naming schema, remove 'In'.